com.fugro.roames.displaz.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "com.fugro.roames.displaz",
+  "branch": "master",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "displaz",
+  "tags": [
+    "master"
+  ],
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "displaz",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/nigels-com/displaz.git",
```

Comment:
Why do you use your fork instead of the official repository?
If you have made changes to the AppData, you should put this file here.
The same applies to the desktop file.

Reply:
I'm still working to have patches accepted upstream.